css/fonts.css

```diff
@@ -1,15 +1,23 @@
 @font-face {
 	font-family: 'Gotham Pro';
-	src: url('GothamPro/GothamPro-Medium.woff') format("woff"),
-	url('GothamPro/GothamPro-Medium.woff2') format("woff2");
-	font-style: 500;
+	src: url('GothamPro/GothamPro-Medium.woff') format("woff"), 
```

Comment:
Codacy found an issue: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/gh/WebHeroSchool/axi/pullRequest?prid=6140750)